modules/profile/files/jenkins/master/configure_git_user.groovy

```diff
@@ -0,0 +1,12 @@
+/* Configure Git User */
+
+/* This script is designed to be run via jenkins-cli.jar in order to update the
+ * configuration for the Jenkins GitSCM plugin.
+ */
+import jenkins.model.Jenkins
+
+def gitscm_config = jenkins.model.Jenkins.getInstance().getDescriptor("hudson.plugins.git.GitSCM")
+gitscm_config.setCreateAccountBasedOnEmail(false)
+gitscm_config.setGlobalConfigName("jenkins")
+gitscm_config.setGlobalConfigEmail("jenkins@build.ros.org")
```

Comment:
This is hard coded here, but since it should be read only, I guess it doesn't really matter.

Reply:
If a willing contributor turned this into a template I would merrily accept. Because the email subsystem isn't part of config management this is fairly benign at the moment.